examples/wasm-example/src/main.rs

```diff
@@ -1,140 +1,59 @@
-use futures::{self, FutureExt};
-
+use routes::signing::SigningExamplesComponent;
 use yew::prelude::*;
+use yew_router::prelude::*;
+
+use crate::routes::fetching::FetchingExamplesComponent;
+mod routes;
 mod services;
 
-fn main() {
-    yew::Renderer::<SubxtExamplesComponent>::new().render();
+#[derive(Routable, PartialEq, Eq, Clone, Debug)]
+pub enum Route {
+    #[at("/fetching")]
+    Fetching,
+    #[at("/signing")]
```

Comment:
Yew is quite nice really; I like the built in browser history/routing stuff, and actually I generally find it quite easy to follow overall!

I wonder how the other frameworks differ; I should really try some of them at some point!